packages/@dcl/inspector/src/lib/logic/config.ts

```diff
@@ -16,6 +17,10 @@ export const CONTENT_URL = version.includes('commit')
   ? 'https://builder-items.decentraland.zone'
   : 'https://builder-items.decentraland.org'
 
+export const SEGMENT_KEY = version.includes('commit')
+  ? 'RmfAUVW7o1EEuvYJGqmVjK9q48BUpnCr'
+  : '8PrGcB0XsogabYgo02gXqfqVM20KKzva'
```

Comment:
we should not have these defaults here, they should be passed from the parent app (builder or vscode), and if none is provided, the tracking should be disabled.